python-code/set-sample/set-sample1.py

```diff
@@ -1,2 +1,3 @@
+print("This is a test file")   
 import os
-import math
```

Comment:
This is a  Unused import os
```suggestion 
```

 from our GitHub Enhacement Bot